cmd/starter/c/starter.c

```diff
@@ -226,7 +226,7 @@ static int apply_container_privileges(struct privileges *privileges) {
      * and to set ambient capabilities. We can't use capset before changing uid/gid
      * because CAP_SETUID/CAP_SETGID could be already dropped
      */
-    if ( prctl(PR_SET_SECUREBITS, SECBIT_NO_SETUID_FIXUP|SECBIT_NO_SETUID_FIXUP_LOCKED) < 0 ) {
+    if ( prctl(PR_SET_SECUREBITS, SECBIT_KEEP_CAPS) < 0 ) {
```

Comment:
Do we need KEEP_CAPS_LOCKED like we had _LOCKED for the FIXUP?

Reply:
No it was a misuse, we need to keep the bit only for the runtime stage 2 and reset it automatically after `execve`, it's only required for the switch of container UID/GID in this function to manipulate capabilities without getting a permission denied error.